app/controllers/iep_documents_controller.rb

```diff
@@ -0,0 +1,37 @@
+class IepDocumentsController < ApplicationController
+
+  before_action :find_iep_document, :authorize!
```

Comment:
This is probably quibbling, but this seems indirect versus just making a method call in `show`.